scripts/linters/pre_commit_linter.py

```diff
@@ -230,13 +230,15 @@ def _lint_all_files(
     css_file_extension_type = 'css' in file_extensions_to_lint
     other_file_extension_type = 'other' in file_extensions_to_lint
 
-    if not other_file_extension_type:
-        general_files_to_lint = '.' + file_extensions_to_lint
+    if js_ts_file_extension_type:
+        general_files_to_lint = _FILES['.js'] + _FILES['.ts']
+    if other_file_extension_type:
+        general_files_to_lint = _FILES['other']
     else:
-        general_files_to_lint = file_extensions_to_lint
+        general_files_to_lint = _FILES['.%s' % file_extensions_to_lint]
```

Comment:
Why don't we store the file extension as a key for the _FILES? (we can avoid this kinda computation, right?)

Reply:
Done.

Reply:
Changed this as this was before because we are getting '.py', '.js' etc as extensions [here](https://github.com/oppia/oppia/blob/bf3309129033ec7ca558495947a06dc6f713475a/scripts/linters/pre_commit_linter.py#L417). I think it is better to do this computation just once here than thousands time inside a for loop. What do you think?